src/Cli/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -88,6 +93,55 @@ public int Execute()
             }
         }
 
+        private static void AttemptToLaunchBrowser(ProjectLaunchSettingsModel launchSettings)
```

Comment:
Have we verified this behavior matches what Visual Studio does WRT to the `applicationUrl` and `launchUrl` settings?

Reply:
Good question! And honestly didn't cross my mind until now.

I work on Linux at home so I'll take a look when I'm in office tomorrow though and see if they align with this behaviour then. :)

Cheers!